src/vs/workbench/contrib/notebook/test/browser/notebookExecutionStateService.test.ts

```diff
@@ -34,6 +34,10 @@ suite('NotebookExecutionStateService', () => {
 	let disposables: DisposableStore;
 	let testNotebookModel: NotebookTextModel | undefined;
 
+	teardown(() => {
```

Comment:
Instead of disposables as a separate variable, `ensureNoDisposablesAreLeakedInTestSuite` returns a disposable store to use. Using that function would also render this teardown section unnecessary.